config/types/files.go

```diff
@@ -77,11 +78,22 @@ type Link struct {
 	Target     string    `yaml:"target"`
 }
 
+func intToPtr(i int) *int {
```

Comment:
Do we want to break this out into a util library?

Reply:
Honestly I find myself implementing so much dumb stuff like this all over the place I'm tempted to make a "missing from stdlib" library for us.

But until I snap and do that, sure.

Reply:
Updated